toptal/picasso
# chore(button): use normal font weight for loading buttons

[FX-1675]

### Description

Use normal font weight for iconless action buttons in loading state

### How to test

- Open Button storybook
- Go to action buttons 
- Notice that iconless action buttons in loading state use normal font weight

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
|<img width="93" alt="Screen Shot 2021-01-28 at 8 21 22 pm" src="https://user-images.githubusercontent.com/77451815/106174732-6b718800-61a6-11eb-845d-d6acb2100ca8.png">|<img width="97" alt="Screen Shot 2021-01-28 at 8 20 21 pm" src="https://user-images.githubusercontent.com/77451815/106174771-73c9c300-61a6-11eb-90fa-4f59e89f5b94.png">|

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[FX-1675]: https://toptal-core.atlassian.net/browse/FX-1675
